### DISCUSSION TITLE
using Bluetooth for nearby device scanning and wifi for file transfer (like google's nearby share)

### DISCUSSION THREAD

**Discussion by GittBittu** [localsend/localsend#540] [Discussion (English, International)]
Nearby share scans for nearby devices with Bluetooth, sends file sharing request the other device and the device automatically connects to your hotspot and starts sharing. it will be a much better and seamless experience if this is implemented in localsend.

**Comment by Bronts** [comment]
In my company, desktop office computers are only connected to the corporate Intranet, and wireless cards are not allowed, but separate laptops are allowed to access the Internet to look up some documents. Sometimes it is necessary to transfer some documents under a few hundred KB between the laptop and the desktop computer, and the use of the USB disk is more troublesome. So using Bluetooth should be more convenient. In addition, Windows built-in Bluetooth transfer can be used for file transfer, but the operation is cumbersome. It is hoped that the LocalSend team will fully consider such use cases.

**Comment by SweetnsourOfficial** [comment]
Isn't nearby share open source?
